lighthouse-cli/printer.js

```diff
@@ -81,7 +81,7 @@ function createOutput(results, outputMode) {
 
   // JSON report.
   if (outputMode === 'json') {
-    return JSON.stringify({audits: results.audits, aggregations: results.aggregations}, null, 2);
+    return JSON.stringify(results, null, 2);
```

Comment:
lgtm on that. 

can you confirm we have test coverage for the structure of the JSON output? 
does the coverage extend to include the CLI/printer code?